numaprom/udsink/train_rollout.py

```diff
@@ -58,14 +59,14 @@ def clean_data(uuid: str, df: pd.DataFrame, hash_col: str, limit=12) -> pd.DataF
     return df
 
 
-def _train_model(uuid, x, anomaly_model):
+def _train_model(uuid, x, model_info, trainer_cfg):
     _start_train = time.perf_counter()
 
-    win_size = anomaly_model.conf["seq_len"]
+    win_size = model_info.conf["seq_len"]
     dataset = StreamingDataset(x, win_size)
```

Comment:
Can you use ModelFactory here? https://github.com/numaproj/numalogic/blob/main/numalogic/config/factory.py#L69